FLT/NumberField/AdeleRing.lean

```diff
@@ -58,12 +83,41 @@ instance : IsModuleTopology (𝔸 K) (L ⊗[K] 𝔸 K) := ⟨rfl⟩
 instance instPiIsModuleTopology : IsModuleTopology (𝔸 K) (Fin (Module.finrank K L) → 𝔸 K) :=
   IsModuleTopology.instPi
 
-open IsDedekindDomain in
+instance instBaseChangeIsModuleTopology : IsModuleTopology (𝔸 K) (𝔸 L) := by
+  sorry
```

Comment:
My plan here is to show that this follows from the corresponding statement at the finite and infinite places, and that the claim at the finite places follows from the fact that A_L^infty is isomorphic (both topologically and algebraically) to the restricted product over the finite places v of *K* of (prod_{w|v} L_w) with respect to (prod_{w|v} O_w) and then to say that prod_{w|v}L_w has the K_v-module topology, prod_{w|v}O_w has the O_v-module topology, and then hope that there's a general theorem about module topologies of restricted products.